source/CardGames.py

```diff
@@ -250,6 +250,14 @@ def rewardPot(self, player):
     player.addMoney(self.money)
     self.money = 0
 
+def checkIfEasterEgg(player):
+  for c in player.hand:
+    if(c.suit == "Hearts" and c.value == 11): #ace of hearts
+      import webbrowser
+      webbrowser.open('https://www.youtube.com/watch?v=V-_O7nl0Ii0')
+      return True
+  return False
+
```

Comment:
well done with this function. We will elaborate on this in later sprints if we have time. Also the if statement parentheses are fine but not necessary.

Reply:
fair enough. parentheses don't matter. this functionality could be expanded into anything.